MIGRATING.md

```diff
@@ -4,11 +4,7 @@
 
 The client no longer normalizes unicode strings in path parameters. This may affect
 some applications using multibyte strings that were previously normalized.:
-To restore the previous behavior, set the following option prior to creating a service.
-
-```ruby
-ClientOptions.default.normalize_unicode = true
-```
+To restore the previous behavior, use the `normalize_unicode` request option.
```

Comment:
Thanks!

Nit:To keep the record in MIGRATING.md straight - is it possible to add a note with a version when it changed to a request option, instead of rewriting history?

Reply:
Ah, fair enough. Thanks for looking into it! :)